x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/select_logs.tsx

```diff
@@ -0,0 +1,241 @@
+                  {
+                    defaultMessage:
+                      'Collect JSON data from listening HTTP port.',
+                  }
+                )}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="m" />
+        </LogsTypeSection>
+        <EuiSpacer size="m" />
+        <EuiLink
+          href="#"
+          target="_blank"
+          onClick={(event: MouseEvent) => {
+            event.preventDefault();
+            navigateToAppUrl('/integrations/browse/observability');
+          }}
+        >
+          Explore other integrations
+        </EuiLink>
+        <EuiHorizontalRule margin="l" />
+        <LogsTypeSection title="Other">
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.uploadLogFiles',
+                  {
+                    defaultMessage: 'Upload log files',
+                  }
+                )}
+                iconType="exportAction"
+                onClick={() => {}}
+                isSelected={false}
+                description={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.uploadLogFiles.description',
+                  {
+                    defaultMessage:
+                      'Upload data from a CSV, TSV, JSON or other log file type for analysis.',
+                  }
+                )}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <OptionCard
+                title={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.useOwnShipper',
+                  {
+                    defaultMessage: 'Get an API key',
+                  }
+                )}
+                iconType="package"
+                onClick={() => {}}
+                isSelected={false}
+                description={i18n.translate(
+                  'xpack.observability_onboarding.selectLogs.useOwnShipper.description',
+                  {
+                    defaultMessage:
+                      'Use your own shipper to collect logs data by generating an API key.',
+                  }
+                )}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </LogsTypeSection>
+      </StepPanelContent>
+      <StepPanelFooter
+        items={[
+          <EuiButton color="ghost" fill onClick={onBack}>
+            {i18n.translate('xpack.observability_onboarding.steps.back', {
+              defaultMessage: 'Back',
+            })}
+          </EuiButton>,
+          <></>,
+        ]}
+      />
+    </StepPanel>
+  );
+}
+
+function LogsTypeSection({
+  title,
+  children,
+}: PropsWithChildren<{ title: string }>) {
+  return (
+    <>
+      <EuiTitle size="s">
+        <h3>{title}</h3>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      {children}
+    </>
+  );
+}
+
+function OptionCard({
```

Comment:
once more of these steps get merged, we will eventually want to used shared components for these cards and sections.